lte/gateway/python/magma/subscriberdb/tests/client_tests.py

```diff
@@ -288,6 +300,14 @@ def get_all_subscribers(self):
             ),
         ]
 
+    @ unittest.mock.patch(
+        'magma.common.service_registry.ServiceRegistry.get_rpc_channel',
+    )
+    def list_suci_profiles(self):
+        return [
```

Comment:
this should only be returning `SuciProfileList` and not a list of `SuciProfileList` right? what is the purpose of this function? I don't see it being used on the tests

Reply:
yes @ardzoht , I resolved it.